visma/io/checks.py

```diff
@@ -82,7 +82,7 @@ def isVariable(term):
 
 def isFunction(term):
 
-    if term in funcs:
+    if term in funcs or term in funcSyms:
```

Comment:
The `term` itself wont be in `funcSyms` but its `symToken` can be present in `funcSyms` (see `tokenizeSymbols(terms)` visma/io/tokenize.py), so the above change wont affect anything.
Also please make the PR to the **dev** branch in future.

Reply:
ok  sorry for that

Reply:
No problem :)